syntax = "proto3";

package delivery;

message DeliveryStatus {
  string order_id = 1;
  string customer_name = 2;
  string delivery_address = 3;
  string status = 4;
  string timestamp = 5;
}
